shell/src/Luna/Shell.hs

```diff
@@ -6,6 +6,10 @@ module Luna.Shell where
 
 import           Luna.Prelude        hiding (String, seq, cons, Constructor)
 import qualified Luna.Prelude        as P
+import           Control.Concurrent  (threadDelay)
+import           Control.Concurrent.Async (race_)
+import           Control.Concurrent.STM
```

Comment:
Could you please enumerate things youre using from this import or use qualified import instead? I know we used to code in such style, but lets try to keep imports either qualified or explicitly listed.